app/gui2/src/components/GraphNode.vue

```diff
@@ -294,67 +298,62 @@ function updatePreprocessor(
   expression: string,
   ...positionalArgumentsExpressions: string[]
 ) {
-  preprocessor.value = { visualizationModule, expression, positionalArgumentsExpressions }
+  visPreprocessor.value = { visualizationModule, expression, positionalArgumentsExpressions }
 }
 
 function switchToDefaultPreprocessor() {
-  preprocessor.value = undefined
+  visPreprocessor.value = DEFAULT_VISUALIZATION_CONFIGURATION
+}
+
+function setVisualizationVisible(visible: boolean) {
+  emit('setVisualizationVisible', visible)
+}
+
+function setVisualizationIdentifier(id: VisualizationIdentifier) {
+  emit('setVisualizationId', id)
 }
```

Comment:
idk whether a wrapper function that only contains a single function call and nothing else, is all that useful